doc/api/pyplot_summary.rst

```diff
@@ -1,29 +1,283 @@
-Pyplot function overview
-------------------------
+******
+pyplot
+******
 
-.. currentmodule:: matplotlib
+.. currentmodule:: matplotlib.pyplot
+
+.. automodule:: matplotlib.pyplot
+    :no-members:
+
+Information
+===========
 
 .. autosummary::
-   :toctree: _as_gen
-   :template: autofunctions.rst
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
 
-   pyplot
+    colormaps
+    get_plot_commands
 
+Global functions
+================
 
-.. currentmodule:: matplotlib.pyplot
+.. autosummary::
+    :toctree: _as_gen
+    :template: autosummary.rst
+    :nosignatures:
 
-.. autofunction:: plotting
+    findobj
+    install_repl_displayhook
+    ioff
+    ion
+    isinteractive
+    pause
+    rc
+    rc_context
+    rcdefaults
+    show
+    switch_backend
+    uninstall_repl_displayhook
 
+Any artist
```

Comment:
What does this mean? Functions that can be used by any artist? Or functions that are applied to every artist on the plot?

Reply:
They are functions that can be applied to every artist. The name was just taken from the`pyplot` file.